### PR TITLE
Use openshift-knative org in udpate-to-head.sh

### PR DESCRIPTION
**Pull request by mgencur**

### CODE REVIEW

File: openshift/release/update-to-head.sh [hunk 1]
```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env bash
 
-# Synchs the release-next branch to main and then triggers CI
+# Synchs the REPO_BRANCH branch to main and then triggers CI
 # Usage: update-to-head.sh
 
 set -e
 REPO_NAME=$(basename $(git rev-parse --show-toplevel))
+REPO_OWNER_NAME="openshift-knative"
```
Comment (skonto) [inline openshift/release/update-to-head.sh:8]: Is this more of a refactoring?

Comment (mgencur) [reply openshift/release/update-to-head.sh:8]: No. It's just a few more lines that are changed to align this with other scripts, e.g. eventing-kafka-broker: https://github.com/openshift-knative/eventing-kafka-broker/blob/main/openshift/release/update-to-head.sh
The main part is changing the "org" in the `hub pull-request` command at the end.

File: openshift/release/update-to-head.sh [hunk 2]
```diff
@@ -32,21 +35,21 @@ make RELEASE=ci generate-release
...
-git commit -m ":robot: Triggering CI on branch 'release-next' after synching to upstream/main"
-git push -f openshift release-next-ci
+git commit -m ":robot: Triggering CI on branch '${REPO_BRANCH}' after synching to upstream/main"
+git push -f openshift ${REPO_BRANCH_CI}
 
 if hash hub 2>/dev/null; then
    # Test if there is already a sync PR in 
-   COUNT=$(hub api -H "Accept: application/vnd.github.v3+json" repos/openshift/${REPO_NAME}/pulls --flat \
-    | grep -c ":robot: Triggering CI on branch 'release-next' after synching to upstream/main") || true
+   COUNT=$(hub api -H "Accept: application/vnd.github.v3+json" repos/${REPO_OWNER_NAME}/${REPO_NAME}/pulls --flat \
```
Comment (skonto) [inline openshift/release/update-to-head.sh:49]: I guess that was the broken part. Are there any other such refs in this repo?

**Pull request merged**